web/modules/custom/rsvplist/rsvplist.permissions.yml

```diff
@@ -0,0 +1,10 @@
+View RSVP List:
```

Comment:
I think it is preferable to write the identifier in the lowercase cause in the Drupal core module lowercase convention is used.

Reply:
@Vishalprasad28 ?